---
title: "About Me"
author: "Luz Zamudio"
output:
  pdf_document:
    toc: yes
  word_document:
    toc: yes
  html_document:
    toc: yes
bibliography: references.bib
---

# Who I am and where I came from

I have spent my whole life in Mexico City, where I was born. Until now, my favorite stage of my life was my childhood. I used to spend my time playing with my two older sisters, imitating mom in everything, going with dad to see him playing basketball, and homework ... lot of homework. But one of my favorits was to visit the zoo, that was pretty close to my house. There, I discover my fascination with animals. 

## Education

I studied elementary, middle and high school in the [IMP](https://www.fundacionmierypesado.org.mx/centros-educativos/instituto/conocenos-2/). When it was time to start the college, I was completely unenthusiastic about it. I felt I was so tired to continue that I did not want to study anything. 

I ended up studying Engineering in Biochemistry at [TESE](http://www.tese.edu.mx/tese2010/). I graduated with honors, but I was not happy with that degree. So, I talked to a close teacher about my main interests (biology, animals, evolution), and told me about the postgraduate programs at [UNAM](https://www.unam.mx/). I concerted a lot of interviews with researchers there, and finally was accepted in one project about the "Evolution of Hummingbirds" leaded by Dra. Blanca Hernández. That is how I started and completed my masters and PhD studies at the [MZFC](http://mzfcaves.fciencias.unam.mx/05posgrads/index.html#parentVerticalTab3). 

But again, at the end of my PhD I was ran out of energy. I look for many posdoctorate options, but I was not lucky. Until last year, I had the priceless opportunity to enroll into the [birdgenoscape project](https://www.birdgenoscape.org/), with a mexican scholarship. It has been challenging but I love it!

## Hobbies

I am not an expert in any of the following things, but I enjoy: 

- Drawing and painting.
- Swimming.
- Hiking, climbing, mountanering.
- Scuba diving.
- Visiting new places.  


In order of apearance: me diving at Cozumel; me and my team at the top of Mt. Iztaccihuatl; visiting Yosemite; climbing at Peña de Bernal.

```{r, echo = FALSE, out.width="50%"}
myimages<-list.files("images/", pattern = ".jpg", full.names = TRUE)
knitr::include_graphics(myimages, auto_pdf = TRUE)
```




# Research Interests

I am interested in the study of the forces that promote speciation, in the evolution, systematics and taxonomy of birds, specially hummingbirds. 

## Influential papers

The time I was enrolling into my masters, the top study about hummingbirds evolution was leaded by Jimmy McGuire at UC Berkeley. The results in [McGuire et al. (2007)](https://academic.oup.com/sysbio/article/56/5/837/1697782?login=true), showed the most robust phylogeny of Trochilidae family (hummingbirs). But, most of the Mesoamerican species hadn't been included, becoming this one of the main objectives of our lab. We focused on studying the phylogenetic relationships on mexican species, and also to study genetic variation at intraspecific level. 

During my PhD, I worked on one widely distributed species (from northern Mexico to Costa Rica), previously known as the Magnificent Hummingbird *Eugenes fulgens*. We found that this complex was conformed by at least two independent lineages, and that the populations distributed in Panama and Costa Rica should be considered as different species [(Zamudio-Beltrán and Hernández-Baños, 2015)](https://www.sciencedirect.com/science/article/abs/pii/S1055790315001268). This study was taking into account by the American Ornithological Society (included some other evidences), to split this complex into two species: the Rivoli's Hummingbird *Eugenes fulgens*, and the Talamanca Hummingbird *Eugenes spectabilis* [(AOS 58 supplement, 2017)](https://academic.oup.com/auk/article/134/3/751/5149324). 

## The mathematics behind my research

Related to the study I talked above, to support the hypothesis that *Eugenes fulgens* complex was conformed by at least two species, I needed to perform an analysis based on coalescence theory, for that I used a BPP approach. This is a Bayesian Markov chain Monte Carlo program for analyzing DNA sequence alignments under the Multispecies coalescent model (MSC) [(Rannala and Yang, 2003)](https://academic.oup.com/genetics/article/164/4/1645/6050371?login=true). 

This method estimates the posterior distribution for species delimitation models, assuming different numbers of species. Each putative species is composed by three key parameters: $\theta$ (the product of effective population size $N$ and mutation rate $\mu$ per site), $\tau_A$ (the time at which the species arose) and $\tau_D$ (the time at which the species splits into two descendent species). The joint posterior distribution of species delimitations and species tree is: 
$$
f(S,\varLambda|D)= \frac{1}{f(D)}f(D|S)f(S|\varLambda)f(\varLambda)
$$
where $S$ denotes the species trees (and therefore $\theta$, $\tau_A$, and $\tau_D$), $\Lambda$ denotes the species delimitation models and $D$ represents the multilocus data. 


## My computing experience

My experience in programming is limited. I started with basic commands in linux when I started performing Bayesian analysis for phylogenetics inferences during the masters. Two years ago, I attended to a workshop about reproducibility and bioinformatics, this was a great course, but as I had no data to analyze I couldn't take too much advantage of this.  

In that workshop we did some basic things like downloading some genetic data like this: 

```sh
#!/bin/bash

# Create working directory
mkdir -p AMINOACIDOS
cd AMINOACIDOS

# Download three sequences from the same protein from different species 
for i in YP_009342035.1 AIM45247.1 ADL09111.1; do
curl -p 
"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=protein&rettype=fasta&id=$i" > 
ej_proteina$i.fas
grep -oE ">\w+.." ej_proteina$i.fas >> out_aminoacidos.txt

# Removing header, looking and counting for AA
grep -v ">" ej_proteina$i.fas > no_header$i.txt
grep -oE "\w" no_header$i.txt | wc -l >> out_aminoacidos.txt
done 
cd ..
```

For sure, I need to excersice my programming skills!

## What I hope to get out of this class

I hope that I will:

- Feel more comfortable using the most popular bioinformatic tools as: R, RStudio, git, gitHub, etc.  
- Learn to make scripts.
- Learn shortcuts or tricks with practice.  
- Improve my listening and speaking English skills.
- Have a lot of fun. 

# Evaluating some R code

I will visualize environmental space from the geographic distribution of Cinammon Hummingbird *Amazilia rutila* compared to 1000 randon points in Mexico. 

```{r, message=FALSE, warning=FALSE}
library(raster)
library(dismo)
library(ggplot2)

rutila<-read.csv(file="data/rutila.csv", sep = ",", header = T)
BIOS <- stack(list.files(path="data/", pattern = "*.asc$",full.names = T)) 

rpoints<-randomPoints(BIOS, 10000, rutila, excludep=TRUE, prob=FALSE, 
	cellnumbers=FALSE, tryf=3, warn=2, lonlatCorrection=TRUE)

# extract values 
BIOS_val <- extract(BIOS, rpoints)
rutila_val <- extract(BIOS, rutila[2:3])

# Plots from environmental and geographic space
BIOS_val <- as.data.frame(BIOS_val)
rutila_val <- as.data.frame(rutila_val)

plot(BIOS_val, pch=21, bg="gray", xlab = "Temperature", ylab="Precipitation"); 
points(rutila_val, col="red", pch=21, bg="orange");
title(main = "Environmental space: Cinnamon Hummingbird")

plot(BIOS[[1]], legend=FALSE, xlab= "lon", ylab= "lat"); 
points(rpoints, pch=21, bg="gray", cex=.4);
points(rutila$lon, rutila$lat, col="red", cex=.4, pch=21, bg="orange");
title(main = "Geographic space: Cinnamon Hummingbird")
```

Note: I need to work on figuring out how to manage citations. 